terratest/test/helpers.go

```diff
@@ -47,8 +57,8 @@ func GetIngressIPs(t *testing.T, options *k8s.KubectlOptions, ingressName string
 }
 
 // Dig gets sorted slice of records related to dnsName
-func Dig(t *testing.T, dnsServer string, dnsPort int, dnsName string) ([]string, error) {
-	port := fmt.Sprintf("-p%v", dnsPort)
+func Dig(t *testing.T, dnsServer string, dnsPort string, dnsName string) ([]string, error) {
+	port := fmt.Sprintf("-p%s", dnsPort)
 	dnsServer = fmt.Sprintf("@%s", dnsServer)
 
 	digApp := shell.Command{
```

Comment:
Just a thought, should we rather use `github.com/miekg/dns` instead of shell command? 
Like here https://github.com/AbsaOSS/coredns-crd-plugin/blob/main/terratest/test/helper.go#L35